common/utils/get-tasl-markup.js

```diff
@@ -0,0 +1,43 @@
+import getLicenseInfo from './get-license-info';
+
+function getTitleHtml(title, author, sourceLink) {
+  if (!title) return '';
+
+  const byAuthor = author ? `, ${author}` : '';
+
+  if (sourceLink) {
+    return `<a href="${sourceLink}" property="dc:title" rel="cc:attributionURL">${title}${byAuthor}</a>. `;
+  } else {
+    return `<span property="dc:title">${title}${byAuthor}.</span> `;
+  }
+}
```

Comment:
Could we convert this to JSX while at it?

Reply:
Do you think it's worth it? I think it could be quite fiddly for no real gain.

Reply:
🤷‍♀️

Reply:
We could just move it into the component as it's not used else where as a function?